dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -1067,6 +1067,15 @@
 			clocks = <&rcc STM32_CLOCK_BUS_AHB2 0x00000001>;
 			status = "disabled";
 		};
+
```

Comment:
Not available for all the stm32h7, so be careful when adding the node there.
stm32h723x and stm32h7a3 series have but not the stm32h743 and next

Reply:
Good catch. I have just fixed that.